README.md

```diff
@@ -25,8 +25,10 @@ It also supports command line arguments in the following format:
     * "c" or "channel" - message will be posted to custom channel, with name specified in *channel name argument*
 * channel name argument(`case insensitive`, `optional`) - Specifies the channel name for custom channel. Only applicable for *custom channel type*, for other channel types will be ignored.
     * *specify your channel name or number* - provide a channel where you want the message to be posted. E.g. "3", "pvp3", "ACMEGuidHeal"
-* excluded players argument(`case insensitive`, `optional`) - Specifies the channel name for custom channel. Only applicable for *custom channel type*, for other channel types will be ignored.
+* excluded players argument(`case insensitive`, `optional`) - Excluding players from buffduty.
     * *`e{<player1>,<player2>,..}`* - provide a list of players you want to exclude from BuffDuty. Please note that players are **comma separated**, no spaces in between. E.g. `e{putris,spaceBag}`.
+* ordered players argument(`case insensitive`, `optional`) - Order list, similar to exclude, to allow priority for who will be assigned a 2nd/etc.
```

Comment:
On reflection I'm not sure 'order' is quite the right word, it's more additional dutes / extra responsibility.. e.g. - Order list, allow priority of who will be given additional buffing duties.

Reply:
Done